staff/felipe-fernandez/array-functions/version-testarude/for-each-jaume-test.js

```diff
@@ -0,0 +1,63 @@
+console.log('TEST forEach');
+
+// var array = ['a', 'b', 'c'];
+
+// var outputs = [];
+
+// forEach(array, function (element, index, array) {    
+//     outputs.push([element, index, array]);
+// });
+
+// check(outputs, [
+//     ['a', 0, ['a', 'b', 'c']],
+//     ['b', 1, ['a', 'b', 'c']],
+//     ['c', 2, ['a', 'b', 'c']]
+// ]);
+
+
+// array = [1, 2, 3];
+
+// var result = 0;
+
+// forEach(array, function (elementValue) {
+//     result = result + elementValue;
+// });
+
+
+// check(result, 6);
+
+
+// // case: no arguments
+
+try {
+    forEach();
+} catch(error) {
+    // check(error instanceof TypeError, true);
+    // check(error.message, 'missing argument 0 when calling function forEach');
+    alert(error.message);
```

Comment:
Dont upload extra files, just function.js and function.test.js, please check your extra files and dont upload commented code